earth.txt

```diff
@@ -0,0 +1 @@
+This could be perfect to set up a base
```

Comment:
Could elaborate more. 
```suggestion
This could be perfect to set up a base. 
Pros: there are many humans to eat there. Has a moon.
Cons: too sunny in the morning for a vampire. 
```